pkg/client/client.go

```diff
@@ -258,12 +259,24 @@ func parseProfile(reader io.ReadCloser) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to unmarshal a response: %s", err)
 	}
-
+	profileFavoriteChoice := 0 
 	if profiles.Type == "VPN" {
-		if len(profiles.Favorites) != 1 {
-			return "", fmt.Errorf("multiple VPN profiles found (FIXME)")
+		if( len(profiles.Favorites) != 1) {
+			profileEnvStr := os.Getenv("PROFILE_FAVORITE")
```

Comment:
I suggest to use CLI arguments instead of env variables. `flag` package already can handle uint: https://golang.org/pkg/flag/#Uint

Reply:
great that's easy. I called it profile-index. Open for better suggestions

Reply:
Ok not so easy as I thought, the build fails with the type checking. What am I missing here. I think it's something painfully obvious. It messes up the types of the flags or something.


pkg/client/client.go:409:2: non-bool sel (type int) used as if condition
pkg/client/client.go:568:2: non-bool closeSession (type int) used as if condition

Reply:
Ok resolved it by changing the flag order

Reply:
Would be open for understanding what caused the error :)
I really like this package and already started using it one of my clusters.